Sources/Tracing/Docs.docc/Guides/TraceYourApplication.md

```diff
@@ -440,26 +440,26 @@ Events usually show up in a in a trace view as points on the timeline (note that
 
 Events cannot be "failed" or "successful", that is a property of a ``Span``, and they do not have anything that would be equivalent to a log level. When a trace span is recorded and collected, so will all events related to it. In that sense, events are different from log statements, because one can easily change a logger to include the "debug level" log statements, but technically no such concept exists for events (although you could simulate it with attributes).
 
-### Where (and how) do Baggage and Spans propagate?
+### Where (and how) do ServiceContext and Spans propagate?
 
 ### Integrations
 
 #### Swift-log integration
 
 Swift-log, the logging package for the server ecosystem, offers native integration with task local values using the `Logger/MetadataProvider`, and its primary application is logging tracing metadata values.
 
-The snippet below shows how one can write a metadata provider and manually extract the baggage and associated metadata value to be included in log statements automatically:
+The snippet below shows how one can write a metadata provider and manually extract the context and associated metadata value to be included in log statements automatically:
 
 ```swift
 import Logging
 import Tracing
 
 // Either manually extract "some specific tracer"'s context or such library would already provide it
 let metadataProvider = Logger.MetadataProvider {
-  guard let baggage = Baggage.current else {
+  guard let context = ServiceContext.current else {
     return [:]
   }
-  guard let context = baggage.someSpecificTracerContext else {
+  guard let context = context.someSpecificTracerContext else {
```

Comment:
```suggestion
  guard let tracerContext = context.someSpecificTracerContext else {
```
`context` might be confusing since it clashes with `context` above.

Reply:
fixed thanks